features/step_definitions/test_steps.rb

```diff
@@ -45,8 +46,22 @@
   organization = project_version.project.organization
 
   description.category = Category.where(organization_id: organization.id, name: category_name).first_or_create if category_name
-  raise 'not yet implemented' if tag_names
-  raise 'not yet implemented' if ticket_names
+
+  if tag_names
+    tag_names.split(',').each do |t|
+      tag = create :tag, name: t, organization: organization
```

Comment:
It would be better to use `Tag.where(organization: organization, name: t).first_or_create` so that the step doesn't fail if used for different tests with the same tag names. Same remark for the tickets.